app/uk/gov/hmrc/economiccrimelevyregistration/controllers/LiabilityBeforeCurrentYearController.scala

```diff
@@ -67,7 +74,10 @@ class LiabilityBeforeCurrentYearController @Inject() (
       form
         .bindFromRequest()
         .fold(
-          formWithErrors => Future.successful(BadRequest(view(formWithErrors, mode))),
+          formWithErrors =>
+            Future.successful(
+              BadRequest(view(formWithErrors, mode, s"${EclTaxYear.currentFinancialYear} to ${EclTaxYear.yearDue}"))
```

Comment:
Can `s"${EclTaxYear.currentFinancialYear} to ${EclTaxYear.yearDue}"` be set in the view instead? I don't see why we would need to pass it but might be missing something :)

Reply:
Done :)